targets/cpp-ue4/templates/PlayFabProxy.uplugin.ejs

```diff
@@ -0,0 +1,27 @@
+{
+    "FileVersion": 3,
+    "FriendlyName": "<%- friendlyName %>",
+    "Version": 0,
+    "EngineVersion" : "4.17.0",
+    "VersionName": "<%- sdkVersion %>",
+    "CreatedBy": "PlayFab and Phoenix Labs",
+    "CreatedByURL": "https://playfab.com/",
+    "DocsURL": "https://api.playfab.com/",
+    "SupportURL": "https://community.playfab.com/index.html",
+    "Category": "PlayFab",
+    "Description": "PlayFab Blueprint SDK for Unreal Engine 4. Current API version: <%- sdkVersion %>",
```

Comment:
This is not the blueprint Sdk.

Reply:
Sorry, must've copied that. That is for the Proxy however, which again handles exposing the CPP to blueprint.

Reply:
No worries, don't fix it here, I'll fix it after.